administration/package.json

```diff
@@ -64,8 +67,8 @@
     "@types/react-router-dom": "^5.3.3",
     "@types/resolve": "^1.20.2",
     "@types/styled-components": "^5.1.26",
-    "@typescript-eslint/eslint-plugin": "^8.8.0",
-    "@typescript-eslint/parser": "^8.8.0",
+    "@typescript-eslint/eslint-plugin": "^8.25.0",
+    "@typescript-eslint/parser": "^8.25.0",
```

Comment:
:wrench: These dependencies should not be necessary anymore as typescript-eslint exports these:
https://typescript-eslint.io/packages/typescript-eslint / https://typescript-eslint.io/packages/typescript-eslint/#migrating-from-legacy-eslintrc-configs
```suggestion
```

Reply:
The migrator asked me to install them after the conversion... I will remove them and see if that works.